src/tests.rs

```diff
@@ -1023,3 +1023,51 @@ fn drain_keep_rest() {
 
     assert_eq!(a, SmallVec::<[i32; 3]>::from_slice(&[1i32, 3, 5, 6, 7, 8]));
 }
+
+#[cfg(feature = "get-size")]
```

Comment:
It looks like this test is failing when both the `"get-size"` and `"union"` features are enabled.  You can reproduce this failure by running `cargo test --features get-size,union`.

The test will also fail on non-64-bit platforms.  You can fix both issues by changing this line to something like

```rust
#[cfg(all(feature = "get-size", target_pointer_with = "64", not(feature = "union")))]
```

Reply:
👍 Fixed. & new branch (w. clean history) pushed. I've also added some tests for when the `union` feature is active.